test/e2e/specs/editor/various/block-mover.spec.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'block mover', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'should show block mover when more than one block exists', async ( {
+		editor,
+		page,
+	} ) => {
+		// Create a two blocks on the page.
+		await page.click( '.block-editor-default-block-appender' );
+		await page.keyboard.type( 'First Paragraph' );
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( 'Second Paragraph' );
+
+		// Select a block so the block mover is rendered.
+		await page.focus( 'text=First Paragraph' );
+		await editor.showBlockToolbar();
+		const count = await page.$$eval(
+			'.block-editor-block-mover',
+			( el ) => el.length
+		);
+		expect( count ).toBe( 1 );
+	} );
+
+	test( 'should hide block mover when only one block exists', async ( {
+		editor,
+		page,
+	} ) => {
+		// Create a single block on the page.
+		await page.click( '.block-editor-default-block-appender' );
+		await page.keyboard.type( 'First Paragraph' );
```

Comment:
Similar to the previous comment, this can change to:
```js
await editor.insertBlock( { name: 'core/paragraph', attributes: { content: 'First Paragraph' } );
```

Reply:
Worked on the feedbacks